### DISCUSSION TITLE
Right control key

### DISCUSSION THREAD

**Discussion by bxnn** [ppy/osu#17895] [Ideas] [answered]
Would be really cool if we could use the right ctrl key on the keyboard to do the ctrl + o to open the settings panel in lazer so that u can use only 1 hand to do the shortcut for people with small hands :D

**Comment by peppy** [comment, marked as answer]
This should already work, I don't believe we differentiate between left and right control

**Reply by bxnn** [reply]
idk right ctrl + o doesnt work for me on lazer i have a french layout keyboard maybe that's why?

**Reply by peppy** [reply]
Can you try going into settings > key bindings and hitting right control-o and see what it shows?

**Reply by bxnn** [reply]
oh shit it was set to Lctrl + o thank you! and sorry for that useless thread :o

**Reply by peppy** [reply]
Hmm, the default is agnostic ctrl-o so if you click the little green light it should reset to that.
